website/docs/r/express_route_connection.html.markdown

```diff
@@ -0,0 +1,134 @@
+  location            = azurerm_resource_group.example.location
+}
+
+resource "azurerm_virtual_hub" "example" {
+  name                = "example-vhub"
+  resource_group_name = azurerm_resource_group.example.name
+  location            = azurerm_resource_group.example.location
+  virtual_wan_id      = azurerm_virtual_wan.example.id
+  address_prefix      = "10.0.1.0/24"
+}
+
+resource "azurerm_express_route_gateway" "example" {
+  name                = "example-expressroutegateway"
+  resource_group_name = azurerm_resource_group.example.name
+  location            = azurerm_resource_group.example.location
+  virtual_hub_id      = azurerm_virtual_hub.example.id
+  scale_units         = 1
+}
+
+resource "azurerm_express_route_circuit" "example" {
+  name                  = "example-expressroutecircuit"
+  location              = azurerm_resource_group.example.location
+  resource_group_name   = azurerm_resource_group.example.name
+  service_provider_name = "Equinix"
+  peering_location      = "Silicon Valley"
+  bandwidth_in_mbps     = 50
+
+  sku {
+    tier   = "Standard"
+    family = "MeteredData"
+  }
+
+  allow_classic_operations = false
+}
+
+resource "azurerm_express_route_circuit_peering" "example" {
+  peering_type                  = "AzurePrivatePeering"
+  express_route_circuit_name    = azurerm_express_route_circuit.example.name
+  resource_group_name           = azurerm_resource_group.example.name
+  shared_key                    = "ItsASecret"
+  peer_asn                      = 100
+  primary_peer_address_prefix   = "192.168.1.0/30"
+  secondary_peer_address_prefix = "192.168.2.0/30"
+  vlan_id                       = 100
+}
+
+resource "azurerm_express_route_connection" "example" {
+  name                             = "example-expressrouteconn"
+  express_route_gateway_id         = azurerm_express_route_gateway.example.id
+  express_route_circuit_peering_id = azurerm_express_route_circuit_peering.example.id
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name which should be used for this Express Route Connection. Changing this forces a new resource to be created.
+
+* `express_route_circuit_peering_id` - (Required) The ID of the Express Route Circuit Peering within this Express Route Connection should be created. Changing this forces a new resource to be created.
+
+* `express_route_gateway_id` - (Required) The ID of the Express Route Gateway within this Express Route Connection should be created. Changing this forces a new resource to be created.
+
+* `authorization_key` - (Optional) The authorization key to establish the Express Route Connection.
+
+* `enable_internet_security` - (Optional) Is internet security enabled for this Express Route Connection?
+
+* `routing` - (Optional)  A `routing` block as defined below.
+
+* `routing_weight` - (Optional) The routing weight associated to the Express Route Connection. Valid values are between `0` and `32000`.
```

Comment:
We could only assign one value for this, right?
```suggestion
* `routing_weight` - (Optional) The routing weight associated to the Express Route Connection. Valid value is between `0` and `32000`.
```

Reply:
Updated